packages/training/python/scripts/hf/trajectories_to_hf_dataset.py

```
@@ -0,0 +1,781 @@
+        (context_text, response_text)
+    """
+    observation = step.get("observation", {})
+    action = step.get("action", {})
+    
+    # Build context from observation
+    context_parts = []
+    
+    # Market state
+    market = observation.get("market", {})
+    if market:
+        context_parts.append(f"**Market State:**")
```

Ruff (0.14.13)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)